atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2607,6 +2619,13 @@ public TransactionReadInfo getReadInfo() {
         return keyValueService.getOverallReadInfo();
     }
 
+    public TransactionCommitLockInfo getCommitLockInfo() {
```

Comment:
Since the use is similar to getReadInfo, you probably want this to be part of the ExpectationsAwareTransaction interface (+Override) since it's now part of the specification of the expectations.